index.html

```diff
@@ -105,7 +106,7 @@ <h3 class="my-works-project-title">Tonic</h3>
                     </ul>
                 </section>
                 <section class="project-more">
-                    <button class="works-see-more">See Project</button>
+                    <button class="works-see-more" type="button" id="tonic-project-btn">See Project</button>
                 </section>
             </div>
         </div>
```

Comment:
- [ ] Please make sure to update the main page so that the projects section is created dynamically using the information stored in that JavaScript array. All of the HTML in that section should be created when the page loads as requested in the project requirements.